packages/core/docs/advanced/caching.md

```diff
@@ -1,99 +1,133 @@
 # Caching
```

Comment:
Shouldn't the title be  `Server-Side Cache`  ?

Reply:
Maybe even `Server-Side Output Cache`? As we are caching SSR Output.

Reply:
I changed it to `Server Side Rendering Cache`. IMHO `Output` is implementation details not worth mentioning in the title

Reply:
Cool